### PR TITLE
Pull network info from NETWORK_LIST json in resourceview

### PR DESCRIPTION
**Pull request by cars**
This is a possible solution/workaround to #16 that utilizes the NETWORK_LIST data map as suggested by @hobovirtual. It works for both privileged (Tenant Admin) and non-privileged users, as well as supporting cases where there might be multiple interfaces/NICs.  It doesn't use the ip_address field but fields more similar to the vRA custom properties one would use to specify the settings in a request.   In the case of a blueprint with a component called 'VM' with two NICs the _resource_configuration_ block would look something like: 
```terraform
 resource_configuration = {
    "VM.cpu" = 2
    "VM.ip_address" = ""
    "VM.name" = ""
    "VM.Network0.IPAddress" = ""
    "VM.Network0.MACAddress" = ""
    "VM.Network0.Name" = ""
    "VM.Network1.IPAddress" = ""
    "VM.Network1.MACAddress" = ""
    "VM.Network1.Name" = ""

  }
```

so using  a count of 2 in the resource definition and the following output: 

```terraform
# Or, print the whole thing:
output "Machine_Info" {
  value = vra7_deployment.test.*.resource_configuration
}
```

would yield something like:
```terraform
Machine_Info = [
  {
    "VM.Network0.IPAddress" = "10.0.3.131"
    "VM.Network0.MACAddress" = "00:50:56:84:e5:d3"
    "VM.Network0.Name" = "Public 3"
    "VM.Network1.IPAddress" = "10.0.4.102"
    "VM.Network1.MACAddress" = "00:50:56:84:8c:2e"
    "VM.Network1.Name" = "Public 4"
    "VM.cpu" = "2"
    "VM.ip_address" = ""
    "VM.name" = "K8S0305"
  },
  {
    "VM.Network0.IPAddress" = "10.0.3.130"
    "VM.Network0.MACAddress" = "00:50:56:84:d4:37"
    "VM.Network0.Name" = "Public 3"
    "VM.Network1.IPAddress" = "10.0.4.101"
    "VM.Network1.MACAddress" = "00:50:56:84:57:7e"
    "VM.Network1.Name" = "Public 4"
    "VM.cpu" = "2"
    "VM.ip_address" = ""
    "VM.name" = "K8S0306"
  },
]
```

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -0,0 +1,9 @@
+cd S:\gosrc\src\github.com\cars\terraform-provider-vra7-1
```
Comment (markpeek) [inline build.cmd]: The build.cmd script crept in even thought you had .gitignore

File: go.mod [hunk 2]
```diff
@@ -13,3 +13,5 @@ require (
 	golang.org/x/net v0.0.0-20190503192946-f4e77d36d62c // indirect
 	gopkg.in/jarcoal/httpmock.v1 v1.0.0-20170412085702-cf52904a3cf0
 )
+
+go 1.13
```
Comment (markpeek) [inline go.mod]: Likely don't want to introduce a 1.13 dependency quite yet. Could you remove go.mod?

**Review by markpeek — CHANGES_REQUESTED** [review]
In general this looks good. A couple of tweaks needed. Also, I'd like @Prativa20 to look at this change and approve. Thanks!

**Comment by cars** [comment]
@markpeek  I think I made the changes correctly

**Review by Prativa20 — APPROVED** [review]
Looks good to me!

**Review by markpeek — APPROVED** [review]

**Pull request merged**